src/components/organisms/LanguageSwitcher.js

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import styled from "styled-components"
+import { useTranslation } from "react-i18next"
+import ListItemText from "@material-ui/core/ListItemText"
+
+const LocaleButton = styled(ListItemText)`
+  && {
+    cursor: pointer;
+  }
+`
+
+function LanguageSwitcher(props) {
+  const changeLanguage = lng => {
+    var path = window.location.pathname
```

Comment:
`window` will fails on build
consider using the bottom code from `BottomNav.js`
```
const {
    route: {
      state: { path },
    },
  } = React.useContext(ContextStore)
```